part-spec/sensor.json

```diff
@@ -0,0 +1,240 @@
+                        "digital"
+                    ],
+                    "type": "string"
+                },
+                "outputResolution": {
+                    "description": "output resolution of acceleration measurement",
+                    "comment": "units of bits",
+                    "type": "number"
+                },
+                "interface": {
+                    "description": "interface(s) for communication to host",
+                    "examples": [
+                        "i2c",
+                        "spi",
+                        "analog"
+                    ],
+                    "type": "string"
+                },
+                "bandwidth": {
+                    "description": "bandwidth of acceleration measurement",
+                    "comment": "units of Hz",
+                    "$ref": "definitions.json#/value"
+                },
+                "outputDataRate": {
+                    "description": "Output Data rate (ODR) of a device",
+                    "comment": "units of Hz",
+                    "$ref": "definitions.json#/value"
+                },
+                "rmsNoise": {
+                    "description": "Broadband rms noise of a device",
+                    "comment": "units of milli g-force",
+                    "$ref": "definitions.json#/value"
+                },
+                "spectralNoiseDensity": {
+                    "description": "Spectral noise density of a device",
+                    "comment": "units of milli g-force per square root of Hz (milli-g/rt-Hz)",
+                    "$ref": "definitions.json#/value"
+                },
+                "currentConsumption": {
+                    "description": "current consumption of a device",
+                    "type": "array",
+                    "items": {
+                      "$ref": "definitions.json#/currentConsumption"
+                    }
+                },
+                "package": {
+                    "description": "package size of a device",
+                    "$ref": "definitions.json#/package"
+                },
+                "pins": {
+                    "description": "array of pin objects with associated properties",
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                }
+            }
+        },
+        {
+            "title": "gyro",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "gyroRanges": {
+                    "description": "range of angular speed that gyro can measure",
+                    "examples": [
```

Comment:
I'm not sure these examples are that useful since they're just numbers

Reply:
removed